teamcity/tests/fixtures/metrics.txt

```diff
@@ -0,0 +1,619 @@
+# HELP finishingBuild_buildFinishDelay_milliseconds Time spent since the server received buildFinish from the agent and a moment when the build has been finished by the server
+# TYPE finishingBuild_buildFinishDelay_milliseconds summary
+finishingBuild_buildFinishDelay_milliseconds_count{experimental="true",nodeId="tc-instance-1",} 74.0
+finishingBuild_buildFinishDelay_milliseconds_total{experimental="true",nodeId="tc-instance-1",} 13902.0
+# TYPE full_agent_waiting_time_milliseconds summary
+full_agent_waiting_time_milliseconds_count{experimental="true",image="Linux-Medium",nodeId="tc-instance-1",version="prod-188",} 19.0
+full_agent_waiting_time_milliseconds_total{experimental="true",image="Linux-Medium",nodeId="tc-instance-1",version="prod-188",} 460726.0
+full_agent_waiting_time_milliseconds_count{experimental="true",image="Linux-Small",nodeId="tc-instance-1",version="prod-188",} 55.0
+full_agent_waiting_time_milliseconds_total{experimental="true",image="Linux-Small",nodeId="tc-instance-1",version="prod-188",} 1945550.0
+full_agent_waiting_time_milliseconds_count{experimental="true",image="Windows-Medium",nodeId="tc-instance-1",version="prod-188",} 0.0
+full_agent_waiting_time_milliseconds_total{experimental="true",image="Windows-Medium",nodeId="tc-instance-1",version="prod-188",} 0.0
+full_agent_waiting_time_milliseconds_count{experimental="true",image="Windows-Small",nodeId="tc-instance-1",version="prod-188",} 0.0
+full_agent_waiting_time_milliseconds_total{experimental="true",image="Windows-Small",nodeId="tc-instance-1",version="prod-188",} 0.0
+# TYPE full_agent_waiting_time_quantile_number gauge
+full_agent_waiting_time_quantile_number{experimental="true",image="Linux-Medium",nodeId="tc-instance-1",quantile="0.5",version="prod-188",} 25105.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Linux-Medium",nodeId="tc-instance-1",quantile="0.8",version="prod-188",} 28765.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Linux-Small",nodeId="tc-instance-1",quantile="0.5",version="prod-188",} 14031.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Linux-Small",nodeId="tc-instance-1",quantile="0.8",version="prod-188",} 123226.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Windows-Medium",nodeId="tc-instance-1",quantile="0.5",version="prod-188",} 0.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Windows-Medium",nodeId="tc-instance-1",quantile="0.8",version="prod-188",} 0.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Windows-Small",nodeId="tc-instance-1",quantile="0.5",version="prod-188",} 0.0
+full_agent_waiting_time_quantile_number{experimental="true",image="Windows-Small",nodeId="tc-instance-1",quantile="0.8",version="prod-188",} 0.0
+# HELP http_requests_duration_milliseconds Duration of http requests to the server
+# TYPE http_requests_duration_milliseconds histogram
```

Comment:
Is this metric being skipped over by choice? I saw a few other ones that are in your payload but not being collected.

Reply:
Yeah, the format is not OpenMetrics-compliant, so it's not getting collected currently. TeamCity recently updated this for the upcoming [2022.12 release](https://youtrack.jetbrains.com/issue/TW-77436/Make-prometheus-metrics-adhere-to-OpenMetrics-specification#focus=Comments-27-6620239.0-0) so I'll add these metrics after I do a little bit of testing

Reply:
This metric is included in the check now along with summary metrics!